sympy/physics/mechanics/kane.py

```diff
@@ -55,6 +56,20 @@ class KanesMethod(_Methods):
         Boolean whether the mass matrices and forcing vectors should use the
         explicit form (default) or implicit form for kinematics.
         See the notes for more details.
+    kd_eqs_solver : str, callable
+        Method used to solve the kinematic differential equations. If a string is
+        supplied, it should be a valid method that can be used with the
+        :meth:`~sympy.matrices.matrices.MatrixBase.solve`. If a callable is supplied, it
+        should take two arguments - the ``A`` matrix and the ``rhs``, and solve the
```

Comment:
I'd write `f(A, rhs)` to be clear here.

Reply:
It would also be nice to list the possible strings here so you don't have to hunt for them.

Reply:
Clickable links don't work in the terminal or many IDEs even.

Reply:
`'LU'` is currently the only one, which I would recommend, so it is also mentioned in the notes. I would not recommend using Cholesky, LDL or any of those others. So mentioning them as if you would recommend it, will only make it more difficult and confusing for the user.

Reply:
Yes, Cholesky and LDL shouldn't be used because those are for (semi-)positive definite matrices I think.